### ISSUE TITLE
Social Media Management Contributions 

### ISSUE THREAD

**Issue by Busayo-ojo** [chaoss/chaoss-africa#33]
Do you have a knack for social media? This is your chance to refine those skills while promoting open source. How to get started:

Make a copy of the social media [content calendar](https://docs.google.com/spreadsheets/d/1oBy3BKXHOvB_JHDwCS_BGicaZ2NISbZ5DCvi1q-h4WY/edit?usp=sharing)
Create your captions
To showcase and submit your amazing contributions, use this [form](https://forms.gle/MYLzfKjfWg9huK7y5)

Looking forward to your brilliant ideas 🎉

**Comment by Mayaleeeee** [comment]
Hello @Busayo-ojo 
Can I pick this up? I would like to work on it.

**Comment by Busayo-ojo** [comment]
Hi @Mayaleeeee thank you for your interest, yes you can work on this!

You can make a copy of this [form ](https://docs.google.com/spreadsheets/d/1oBy3BKXHOvB_JHDwCS_BGicaZ2NISbZ5DCvi1q-h4WY/edit?usp=sharing). When you're done please submit with this [form](https://forms.gle/MYLzfKjfWg9huK7y5)

**Comment by Mayaleeeee** [comment]
Thank you @Busayo-ojo 
Do I need to create an account on the Hacktoberfest first website to count as a Hacktoberfest contribution?

**Comment by Busayo-ojo** [comment]
Yes @Mayaleeeee, doing that is great. We will also track contributions from our end

**Comment by Mayaleeeee** [comment]
> Yes @Mayaleeeee, creating an account on the Hacktooberfest website is great. We will also track contributions from our end.
> 

Thank you @Busayo-ojo
